README.old.md

```diff
@@ -0,0 +1 @@
+# Cryptocurrency-App
```

Comment:
**[Optional]**

- [ ] It would be nice if you remove this unnecessary readme.old file from your repository to make your repository clean and look professional 😊